iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -149,8 +147,9 @@ protected void readElement(String id, ShuntCompensatorAdder adder, NetworkXmlRea
                     break;
                 case AliasesXml.ALIAS:
                     IidmXmlUtil.assertMinimumVersion(ROOT_ELEMENT_NAME, AliasesXml.ALIAS, IidmXmlUtil.ErrorMessage.NOT_SUPPORTED, IidmXmlVersion.V_1_3, context);
+                    String aliasType = context.getReader().getAttributeValue(null, "type");
                     String alias = context.getAnonymizer().deanonymizeString(context.getReader().getElementText());
-                    aliases.add(alias);
+                    aliases.put(alias, aliasType);
```

Comment:
Just a thought but it is very dangerous to have only for shunt kind of duplication of alias management. It is because of model sub element?

Reply:
Yes because shunts are the only equipment who needs its sub element to be created (other equipment can be created without their sub elements).